scripts/install_matomo

```diff
@@ -6,18 +6,18 @@
 
 # Parameters: all manadatory and must be prefix with "--" on command line.
 declare -A parameters=( \
-  [data_disk_size]= \
-  [data_disk_mount_point_path]= \
-  [db_server_admin_password]= \
-  [db_server_admin_username]= \
-  [db_server_fqdn]= \
-  [db_server_matomo_database]= \
-  [db_server_matomo_password]= \
-  [db_server_matomo_username]= \
-  [php_max_execution_time]= \
-  [smtp_server_fqdn]= \
-  [smtp_server_private_ip]= \
-  [web_server_fqdn]= \
+  [data_disk_size]=\
+  [data_disk_mount_point_path]=\
```

Comment:
Since the parameters are enclosed in ( ), is the param order important or it can be in alphabetical order?

Reply:
Pas d'importance. Corrigé.